DesignKit/Source/Colors.swift

```diff
@@ -68,4 +68,11 @@ public protocol Colors {
     /// - Avatars default states that include first name letter
     var namesAndAvatars: [ColorType] { get }
     
+    // MARK: - Others colors
+    
+    /// White
+    var white: ColorType { get }
+    
+    /// Purple
+    var purple: ColorType { get }
```

Comment:
The property name should reflect the purpose. It should be something like: 
```suggestion
    var liveLocationIconColor: ColorType { get }
```